src/Search/Console.php

```diff
@@ -120,6 +120,10 @@ public function __construct(Application $console, Kernel $app)
         $this->app = $app;
         $this->root = __DIR__.'/../..';
 
+        if (!defined('GEARMAN_INSTALLED')) {
```

Comment:
Ah this is to aid local development. For example I have 2 PHP instances, a clean one without XDebug and Gearman for running through the tests in > 2s and another with XDebug and Gearman for other tests.

Its found in the test bootstrap.php but does not get defined when testing console commands, it must run it in isolation.